### ISSUE TITLE
Need to login a second time to PILA when trying to create a custom task 

### ISSUE THREAD

**Issue by vs-ranjani** [pila-project/pila-platform#101]
PILA Create asks me to login again when I try to access the customiser tool. It might be nice to skip this step if the teacher is already logged in to PILA. 

![image](https://github.com/pila-project/pila-platform/assets/168413098/410d541e-9d08-4752-b735-d03c6a21a651)

**Comment by vs-ranjani** [comment]
Same with the Karel customiser.

**Comment by OECDPILA** [comment]
(Low Priority) - I think we can keep this until after the workshop.

**Comment by jasonford** [comment]
Agreed we can wait on making updates to this until after the workshop. I think one thing that will help a lot is if we increase the "logged in time" for sessions. right now we are a little aggressive closing sessions and requiring re-log in. Closing for now, but we can bring this up later
